src/locales/en/translation.json

```
@@ -181,8 +183,16 @@
     "observers": "Observers",
     "status": "Status",
     "add": "Add sample unit",
+    "last_edited_by": "Last edited by",
     "submitted_no_data_message": "This project has no submitted sample units.",
```

This key is almost identical to `no_submitted_sample_units`. Can we update both use cases to the following?
```suggestion
    "no_submitted_sample_units": "This project has no submitted sample units.",
```